aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -25,7 +25,7 @@ For more information and configuration guidance, see the following topics:
 
 ## Install ASP.NET Core Module (ANCM)
 
-The ASP.NET Core Module (ANCM) is installed with the .NET Core Runtime from the [.NET Core Hosting Bundle](xref:host-and-deploy/iis/hosting-bundle). The ASP.NET Core Module is forward and backward compatible with LTS releases of .NET.
+The ASP.NET Core Module (ANCM) is installed with the .NET Core Runtime from the [.NET Core Hosting Bundle](xref:host-and-deploy/iis/hosting-bundle). The ASP.NET Core Module is forward and backward compatible with [in-support LTS releases of .NET](https://dotnet.microsoft.com/platform/support/policy/dotnet-core#lifecycle).
```

Comment:
Actually, I'm a bit confused about this one. Why do we specifically call out `LTS` here? Should we just mention `in support` instead?

Reply:
I think I understand why - mainly because STS releases are expected to carry more breaking changes, than the LTS ones.

Reply:
Yes, the product unit said "LTS." I can try and research back to see where it came from.

Reply:
@halter73 what do you think about this?

Reply:
I thought it had to do with their testing ... i.e., they aren't testing STS-to-LTS match-ups between ANCM and apps (or however they test if it isn't apps).

Reply:
Looks like it came in here ...

https://github.com/dotnet/AspNetCore.Docs/pull/21914

Reply:
@mkArtakMSFT ... Here's exactly where the "LTS" remark came from ... Justin said it at ...

https://github.com/dotnet/AspNetCore.Docs/pull/21914#discussion_r603563438

Reply:
Ping @halter73

Reply:
To the contrary, I think we should encourage people to use the latest STS release if they want to. It's still supported. We need more people using our new stuff.

I agree with changing this to just be "in-support releases" instead of "in-support LTS releases". @adityamandaleeka @DamianEdwards.

Reply:
Agree with @halter73, just use the "in-support releases" phrasing.